### PR TITLE
feat(pre-publish script): Add pre-publish shell script to support standard-version prerelease 

### PR DESCRIPTION
**Pull request by iamgarima**
#58

### CODE REVIEW

File: bin-dev-scripts/standard-version-release.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```
Comment (gja) [inline bin-dev-scripts/standard-version-release.sh]: Bash -e to abort on error

File: bin-dev-scripts/standard-version-release.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+npm install
+npm test
```
Comment (gja) [inline bin-dev-scripts/standard-version-release.sh]: Please remove npm test from this. Let that be in pre release only

File: bin-dev-scripts/standard-version-release.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+if [ $BRANCH == 'master' ]
+then
+  npx standard-version
+else
+  npx standard-version --prerelease $BRANCH
+fi
+
+unset BRANCH
+git push --follow-tags origin
```
Comment (gja) [inline bin-dev-scripts/standard-version-release.sh]: Mention which branch you push to

Comment (gja) [reply bin-dev-scripts/standard-version-release.sh]: And please put this file in /bin

File: bin-dev-scripts/standard-version-release.sh [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
```
Comment (amithgeorge) [inline bin-dev-scripts/standard-version-release.sh]: generally, use `#!/usr/bin/env sh` and `set -o nounset -o pipefail -o errexit` . And prefer not to use bash specific syntax in the scripts. I don't see any bash specific syntax, so thats good. Why `sh` and not `bash`, cuz bash is not guaranteed to always be there. `sh` represents whatever is the default shell for the user running the script. I prefer the explicit `set` commands as opposed to the `-e` commandline option for bash.

Comment (amithgeorge) [reply bin-dev-scripts/standard-version-release.sh]: Not a blocker, just mentioning it.

File: bin-dev-scripts/standard-version-release.sh (hunk 3 above)
Comment (amithgeorge) [reply bin-dev-scripts/standard-version-release.sh]: I didn't want the dev-time scripts to be part of the bundled package. So, we can put it in the bin folder and ensure that these files are not bundled into the tarball.

File: bin-dev-scripts/standard-version-release.sh (hunk 4 above)
Comment (iamgarima) [reply bin-dev-scripts/standard-version-release.sh]: Okay, will read on this and update the script.

**Pull request merged**